storybook/src/components/Button/Button.stories.tsx

```diff
@@ -3,27 +3,39 @@
  * Copyright Gemeente Amsterdam
  */
 
-import { Icon } from '@amsterdam/design-system-react'
 import { Button } from '@amsterdam/design-system-react/src'
-import { ShareIcon } from '@amsterdam/design-system-react-icons'
+import * as Icons from '@amsterdam/design-system-react-icons'
 import { Meta, StoryObj } from '@storybook/react'
 
 const meta = {
   title: 'Components/Buttons/Button',
   component: Button,
   args: {
-    children: 'Button label',
-    variant: 'primary',
+    children: 'Versturen',
     disabled: false,
+    variant: 'primary',
+    /* This is the only was storybook will honor the conditional in the iconPosition argType (line 29) */
+    // @ts-ignore
+    icon: null,
```

Comment:
Is this still necessary now we always show the icon controls?

Reply:
No, missed it.